zerver/tests/test_bots.py

```diff
@@ -164,7 +164,7 @@ def test_add_bot(self) -> None:
         self.login("hamlet")
         self.assert_num_bots_equal(0)
         events: List[Mapping[str, Any]] = []
```

Comment:
All of these pre-definitions of `events` are now unnecessary -- that's one of the reasons to switch to the contextmanager returning the list.

Reply:
Makes sense! -- updated.